exercises/luhn/src/example.erl

```diff
@@ -1,5 +1,5 @@
 -module(example).
--export([valid/1, create/1, checksum/1, test_version/0]).
+-export([valid/1, create/1, checksum/1, isNumeric/1, isMinLength/1, test_version/0]).
```

Comment:
After I have asked you to remove the `create_test/0` from the suite, we can remove `create/1` from the list of exports as well. Also it should be okay then to remove that function completely, since it is not used from within the module, is it?